codes.json

```diff
@@ -364,6 +364,102 @@
       "en": "No groups names were provided."
     }
   },
+  "IncorrectNumber": {
+    "status": 402,
```

Comment:
Any reason we use 402 which is not used very ofter, although it says "Payment Required: This code is reserved for future use." ? I suggest we follow return 400 as other invalid request cases